### DISCUSSION TITLE
Can't sleep remote pc

### DISCUSSION THREAD

**Discussion by huxim** [rustdesk/rustdesk#8476] [General]
### Bug Description

Can't sleep remote pc

### How to Reproduce

1.connect to a pc
2.click sleep in start menu
3.the remote pc did not sleep

### Expected Behavior

the remote pc should sleep

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.6 -> 1.2.6

### Screenshots

![PixPin_2024-06-26_10-10-30](https://github.com/rustdesk/rustdesk/assets/2505801/c7f78a28-a0c6-40ef-9428-66ec2e3e442a)

**Comment by rustdesk** [comment]
close to favor https://github.com/rustdesk/rustdesk/discussions/8473#discussioncomment-9891505
